src/_posts/gamestate.md

```diff
@@ -0,0 +1,346 @@
+---
+title: "Managing Application and Game State in A-Frame"
+author: twitter|andgokevin|Kevin Ngo
+date: 2017-4-05
+layout: blog
+image:
+  src: https://cloud.githubusercontent.com/assets/674727/24813155/d1afc7fe-1b81-11e7-99b7-b444babd2593.png
+---
+
+Larger VR applications need a way of cleanly managing application or game
```

Comment:
I'd draw folks in with a bit more examples. what constitutes a "larger VR application"? should I read this if I'm a novice/intermediate Web Dev/VR creator? or this advanced reading?